components/timezone/src/ixdtf.rs

```diff
@@ -391,20 +275,124 @@ impl CustomZonedDateTime<AnyCalendar> {
     ///     metazone_id: Some(MetazoneId(tinystr!(4, "amce"))),
     ///     zone_variant: None,
     /// });
+    /// ```
+    ///
+    /// An IXDTF string can provide a time zone in two parts: the DateTime UTC Offset or the Time Zone
```

Comment:
question: Would it be worthwhile linking to the `ixdtf` crate documentation if anyone wanted to read more about IXDTF strings?

Reply:
Done